sdk/core/core-asynciterator-polyfill/package.json

```diff
@@ -45,14 +47,12 @@
     "lint:fix": "eslint package.json src --ext .ts --fix --fix-type [problem,suggestion]",
     "pack": "npm pack 2>&1",
     "prebuild": "npm run clean",
-    "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
-    "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
-    "test": "npm run build:test && npm run unit-test && npm run integration-test",
+    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
+    "test:node": "npm run clean && tsc -p . && npm run unit-test:node && npm run integration-test:node",
+    "test": "npm run clean && tsc -p . && npm run unit-test:node && rollup -c 2>&1 && npm run unit-test:browser && npm run integration-test",
```

Comment:
instead of `tsc -p .`, can we `npm run build` ? and perhaps if `prebuild` runs in this case, we can omit `clean` after all, since it will happen automatically?

Reply:
We can in this case only since we do not use rollup for this package.

Reply:
Done.